plugins/search-react/api-report.md

```diff
@@ -82,21 +115,596 @@ export type SearchAutocompleteFilterProps = SearchFilterComponentProps & {
 };
 
 // @public
-export const SearchBar: ({ onChange, ...props }: SearchBarProps) => JSX.Element;
+export type SearchAutocompleteProps<
+  T,
+  Multiple extends boolean | undefined,
+  DisableClearable extends boolean | undefined,
+  FreeSolo extends boolean | undefined,
+> = Omit<
+  AutocompleteProps<T, Multiple, DisableClearable, FreeSolo>,
+  'renderInput'
+> & {
+  inputDebounceTime?: SearchBarProps['debounceTime'];
+  renderInput?: (params: AutocompleteRenderInputParams) => JSX.Element;
+};
+
+// @public
+export const SearchBar: React_2.ForwardRefExoticComponent<
+  Pick<
```

Comment:
Maybe discussed elsewhere, but is there any way to ensure this isn't explicitly part of the API report?

Reply:
Yes, I defined the type explicitly rather than letting Typescript infer it. Sounds good to you?